src/main/java/com/dev/cinema/controllers/MovieController.java

```diff
@@ -15,6 +17,7 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
+@Validated
```

Comment:
Why do you use this annotation?

Reply:
I was hoping this annotation works as general for the whole class, but iI am not sure, is it the correct translation. The description says that is for group validation.

Reply:
you can see an example here
https://www.baeldung.com/spring-valid-vs-validated